assets/css/frontend.less

```diff
@@ -1174,4 +1174,47 @@ Menus
 			display: none !important;
 		}
 	}
+}
+
+/*--------------------------------------------------------------
```

Comment:
Make sure that this is the minimum needed CSS code, maybe there's a way to load it from WP without duplicating, if not then ok.